src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -844,20 +856,23 @@ private TaskStatus GetTaskStatus(DatabaseTopology topology, PeriodicBackupConfig
                         NotificationSeverity.Info));
                 }
 
+                responsibleNodeTag = null;
                 return TaskStatus.Disabled;
             }
 
-            var backupStatus = GetBackupStatus(configuration.TaskId);
-            var whoseTaskIsIt = BackupUtils.WhoseTaskIsIt(_serverStore, topology, configuration, backupStatus, _database.NotificationCenter, keepTaskOnOriginalMemberNode: true);
-            if (whoseTaskIsIt == null)
-                return TaskStatus.ClusterDown;
+            responsibleNodeTag = GetResponsibleNodeTag(_serverStore, _database.Name, configuration.TaskId);
+            if (responsibleNodeTag == null)
+            {
+                // the responsible node wasn't set by the cluster observer yet
+                return TaskStatus.Disabled;
```

Comment:
I think this will confuse users to see the backup as disabled?

Reply:
You are right, previously we returned `ClusterDown`.